test/features/event-checkin.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+Feature('Check In Event', () => {
```

Comment:
I cherry-picked a couple of commits from branch feature/IBM-1 to get mocha-cakes 2 ... so got that file in the commit.